src/sprites/rendered-target.js

```diff
@@ -228,6 +228,9 @@ class RenderedTarget extends Target {
         if (this.isStage) {
             return;
         }
+        if (!isFinite(direction)) {
+            return;
+        }
```

Comment:
I think this might be better handled by the `wrapClamp` utility, since we will probably have other cases which will need to handle `Infinity`.  Would you mind moving this into `wrapClamp`?